docs/docusaurus/docs/terms/data_context.md

```diff
@@ -113,8 +113,6 @@ Here's a typical example:
 Running `test_yaml_config()` will show some feedback on the configuration. The helpful output can include any result 
 from the "self check" of an artifact produced using that configuration.  You should note, however, that `test_yaml_config()` never overwrites the underlying configuration.  If you make edits in the course of your work, you will have to explicitly save the configuration before running `test_yaml_config()`.
 
-For more detailed guidance on using the `test_yaml_config()` method, please see our guide on [how to configure DataContext components using test_yaml_config](../guides/setup/configuring_data_contexts/how_to_configure_datacontext_components_using_test_yaml_config.md).
```

Comment:
The entire section `### Interactively testing configurations from your Data Context` is about `test_yaml_config()` -- do we want to remove this full section, instead of just the part with the link to the `test_yaml_config()` guide?

Reply:
good idea